openeogeotrellis/layercatalog.py

```diff
@@ -1045,8 +1045,9 @@ def check_missing_products(
 
         method = check_data.get("method")
         if method == "creo":
-            creo_catalog = CreoCatalogClient(**check_data["creo_catalog"])
-            missing = [p.getProductId() for p in creo_catalog.query_offline(**query_kwargs)]
```

Comment:
maybe also put a TODO comment in the `query_offline` explaining why the implementation needs work

Reply:
Added a comment